gapic/src/main/com/google/gapid/views/DeviceValidationView.java

```diff
@@ -63,26 +78,29 @@ public DeviceValidationView(Composite parent, Models models, Widgets widgets) {
 
     validationPassed = false;
 
-    setLayout(new GridLayout(/* numColumns= */ 2, /* makeColumnsEqualWidth= */ false));
+    setLayout(new GridLayout(/* numColumns= */ 3, /* makeColumnsEqualWidth= */ false));
     setLayoutData(new GridData(SWT.FILL, SWT.TOP, true, false));
 
     // Status icon (loader) & accompanying text
     statusLoader = widgets.loading.createWidgetWithImage(this, 
         widgets.theme.check(), widgets.theme.error());
-    statusLoader.setLayoutData(new GridData(SWT.LEFT, SWT.BOTTOM, false, false));
-    statusText = createLink(this, "", e -> {
-      Program.launch(URLs.DEVICE_COMPATIBILITY_URL);
-    });
-    statusText.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false));
+    statusLoader.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false));
+    statusText = withLayoutData(createLabel(this, ""),
+      new GridData(SWT.FILL, SWT.CENTER, true, false));
+    retryButton = withLayoutData(createButton(this, "Retry", e -> {
+      // Intentionally empty.
+    }), new GridData(SWT.FILL, SWT.CENTER, false, false));
```

Comment:
`CENTER` doesn't make sense, without grabbing the extra space (e.g. if `verticalAlignment` is `CENTER`, `grabExcessVerticalSpace` should be `true`).